src/state/metadata/selectors.ts

```diff
@@ -0,0 +1,33 @@
+import { createSelector} from "reselect";
+
+import { State } from "../";
+
+import { NavigationTab } from "../../constants/index";
+import { getNavTab } from "../selections/selectors";
+
+import {
+    MenuElement,
+    Post,
+    User} from "./types";
```

Comment:
Another tiny stylistic nit: there should be a newline and trailing comma after `User`